sdk/documentintelligence/ai-document-intelligence-rest/src/documentIntelligence.ts

```diff
@@ -40,7 +41,12 @@ export default function createClient(
       logger: options.loggingOptions?.logger ?? logger.info,
     },
     credentials: {
-      scopes: options.credentials?.scopes ?? ["https://cognitiveservices.azure.com/.default"],
+      scopes: (
```

Comment:
is it possible to combine manual code with generated code? otherwise we need manually to maintain this if we will refresh this.

Reply:
this is the last version of this package, we are fine with this in this case.